# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
geometry_msgs/Pose pickup_pose
geometry_msgs/Pose place_pose
